### PR TITLE
correct LightGraphs compability

### PR DESCRIPTION
**Pull request by pszufe**

### CODE REVIEW

File: src/bipartite.jl [hunk 1]
```diff
@@ -72,7 +72,11 @@ end
 
 
 
-LightGraphs.is_directed(b::BipartiteView) = false
+LightGraphs.is_directed(b::BipartiteView{T}) where T = false
+
+LightGraphs.is_directed(::Type{BipartiteView{T}}) where T = false
+
+Base.eltype(::BipartiteView{T}) where T = T
```
Comment (bkamins) [inline src/bipartite.jl]: what is the interpretation of `eltype` in LightGraphs.jl? As I look at:
```
julia> eltype(erdos_renyi(10,0.1))
Int64

julia> eltype(erdos_renyi(UInt8(10),0.1))
UInt8
```
it seems it is the type of the internal storage for vertex numbering. But I might be wrong.

Comment (pszufe) [reply src/bipartite.jl]: It is exactly the type of elements in the adjacency lists of a graph.
In particular it is not edge weight in other libraries such as SimpleWeightedGraphs.jl. 
Hence, in our case the value returned by eltype should be `Int`

**Comment by bkamins** [comment]
Do we implement https://juliagraphs.github.io/LightGraphs.jl/latest/developing.html#Developing-Alternate-Graph-Types-1 for other methods?

**Comment by pszufe** [comment]
> 
> 
> Do we implement https://juliagraphs.github.io/LightGraphs.jl/latest/developing.html#Developing-Alternate-Graph-Types-1 for other methods?

Three methods were missing and have been added

File: src/bipartite.jl [hunk 2]
```diff
@@ -47,6 +47,9 @@ function LightGraphs.has_edge(b::BipartiteView, s, d)
     end
 end
 
+function LightGraphs.has_vertex(b::BipartiteView, v::Integer) 
+   return v >= 1 && v <= LightGraphs.nv(b)
```
Comment (bkamins) [inline src/bipartite.jl]:
```suggestion
   1 <= v <= LightGraphs.nv(b) 
```
and it probably be a one-liner without `function`

File: src/twosection.jl [hunk 3]
```diff
@@ -54,6 +54,10 @@ function LightGraphs.has_edge(t::TwoSectionView, s, d)
     !isempty(intersect(keys(t.h.v2he[s]), keys(t.h.v2he[d])))
 end
 
+function LightGraphs.has_vertex(t::TwoSectionView, v::Integer) 
+   return v >= 1 && v <= LightGraphs.nv(t)
```
Comment (bkamins) [inline src/twosection.jl]: comment as above

**Pull request merged**